zest/releaser/utils.py

```diff
@@ -138,40 +200,40 @@ def ask(question, default=True, exact=False):
     """
     if AUTO_RESPONSE:
         if default is None:
-            msg = ("The question '%s' requires a manual answer, but "
-                   "we're running in --no-input mode.")
-            msg = msg % question
+            msg = (u"The question '{0}' requires a manual answer, but "
+                   u"we're running in --no-input mode.")
+            msg = msg.format(question)
             raise RuntimeError(msg)
-        logger.debug("Auto-responding '%s' to the question below.",
-                     default and "yes" or "no")
+        logger.debug(u"Auto-responding '{0}' to the question below.".format(
+            default and u"yes" or u"no"))
         logger.debug(question)
         return default
-    while True:
-        yn = 'y/n'
+    while 1:
```

Comment:
I prefer `while True:`.  The speed difference is 20 *microseconds* on Python 2 and disappears completely on Python 3, which forbids assignments to `True` and `False`.